### PR TITLE
Remove paragraph about calling {{Specifications}} with a parameter

### PR DESCRIPTION
**Pull request by teoli2003**
This is deprecated, let's not document this as a legit way to use the macro.

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/specification_tables/index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Specification tables
 slug: MDN/Writing_guidelines/Page_structures/Specification_tables
 page-type: mdn-writing-guide
+browser-urls: css.properties.text-align
```
Comment (bsmth) [inline files/en-us/mdn/writing_guidelines/page_structures/specification_tables/index.md:5]: Flaw here still:

>No first query argument or 'browser-compat' or 'spec-urls' front-matter value passed`

```suggestion
browser-compat: css.properties.text-align
```

**Comment by teoli2003** [comment]
Stupid me, I first wrote `spec-urls` than changed it but not completely

**Review by bsmth — APPROVED** [review]
Good fix, tnx

**Comment by hamishwillee** [comment]
@teoli2003 Looks good. Should we also talk about what replaces this usage?

Specifically, most of the time we used that macro with the key was because we wanted to have multiple compat or specification values. The right way to do that now is to add multiple keys in the browser compat, or `spec-urls:` keys. 

Do you want to add something for that or should I?

**Comment by teoli2003** [comment]
Can you write it? I'll happily review it!

**Pull request merged**